frontend/src/queries/schema.ts

```diff
@@ -624,6 +624,8 @@ export interface InsightsQueryBase extends Node {
     aggregation_group_type_index?: integer
     /** Sampling rate */
     samplingFactor?: number | null
+
+    explicitDate?: boolean
```

Comment:
Fly-by comment (off for easter, will have a deeper look next Tuesday) - I don't think we handle this setting in the new insights at all. Seems it gets passed in via the date range filter. I'll look into adding it and would appreciate any context :) After a first look I'd think this should belong into the `dateRange` object?

Reply:
Didn't see the DateRange component before, but agree should go there!

For context,  we need this setting whenever we don't want to round date ranges to start and end of day. This is particularly important for experiments because we want to discard all testing data that came before the exact moment the experiment began.

Reply:
We'll need this setting as well to move both trends & funnel experiments backend off of filters and onto query 😅

Reply:
For more context, the appropriate setting in filters is: https://github.com/PostHog/posthog/blob/master/posthog/schema.py#L2549